yarn-project/foundation/src/wasm/wasm/async_wasm_wrapper.ts

```diff
@@ -8,12 +8,13 @@ export abstract class AsyncWasmWrapper extends WasmWrapper {
   protected asyncCallState = new AsyncCallState();
 
   /**
-   * 20 pages by default. 20*2**16 \> 1mb stack size plus other overheads.
+   * 30 pages by default. 30*2**16 \> 1mb stack size plus other overheads.
    * 8192 maximum by default. 512mb.
    * @param initial - Initial memory pages.
    * @param maximum - Max memory pages.
+   * @returns Self.
    */
-  public async init(initial = 20, maximum = 8192): Promise<this> {
+  public async init(initial = 30, maximum = 8192): Promise<this> {
```

Comment:
Do we still need this change after https://github.com/AztecProtocol/aztec3-packages/pull/263? (same for the two other wasm_wrappers)

Reply:
Ah gg, ill need to rebase